pkg/controller/complianceremediation/complianceremediation_controller.go

```diff
@@ -369,6 +381,116 @@ func (r *ReconcileComplianceRemediation) countKubeUnmetDependencies(rem *compv1a
 	return nMissingDeps, nil
 }
 
+func (r *ReconcileComplianceRemediation) handleValueRequired(rem *compv1alpha1.ComplianceRemediation, logger logr.Logger) (reconcile.Result, error) {
+	annotations := rem.GetAnnotations()
+	labels := rem.GetLabels()
+	if annotations == nil {
+		annotations = make(map[string]string)
+	}
+	requiredValues := annotations[compv1alpha1.RemediationValueRequiredAnnotation]
+	requiredValuesList := strings.Split(requiredValues, ",")
```

Comment:
How do we handle empty strings here? Sounds like something a user may set accidentally.

Reply:
Maybe I should handle the empty string during the parsing?

Reply:
It has been addressed, I will skip all the empty string during the looping